### ISSUE TITLE
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.

### ISSUE THREAD

**Issue by Li-Fanda** [THUDM/LongWriter#13]
### System Info / 系統信息

linux系统
transformers == 4.43.0
torch == 2.4.0
python == 3.12.0

### Information / 问题信息

- [X] The official example scripts / 官方的示例脚本
- [ ] My own modified scripts / 我自己修改的脚本和任务

### Reproduction / 复现过程

在conda环境中进入项目路径并进入搭好环境的虚拟环境，然后运行python trans_web_demo.py；
项目正常运行启动并输出Gradio网址，打开浏览器进入该网页；
在输入框中输入”你是谁？“，点击Submit，发现一直处于推理的过程，再查看conda窗口，发现报错如下：

The attention mask is not set and cannot be inferred from input because pad token is same as eos token.As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.

![1322](https://github.com/user-attachments/assets/d1d343a2-c0aa-4aba-9463-dca18a41e322)

### Expected behavior / 期待表现

能够正常在Gradio网页中进行推理

**Comment by bys0318** [comment]
请更新到最新的模型文件

**Comment by Li-Fanda** [comment]
> 请更新到最新的模型文件，主要是[modeling_chatglm.py](https://huggingface.co/THUDM/LongWriter-glm4-9b/blob/main/modeling_chatglm.py)文件，也同时更新trans_web_demo.py文件

非常感谢！已经推理成功，祝好！
